src/components/tables/RowActions/Delete/Button.tsx

```diff
@@ -14,12 +15,32 @@ interface Props {
 function DeleteButton({ selectableTableStoreName }: Props) {
     const generator = () => null;
 
+    const isCapture =
+        selectableTableStoreName === SelectTableStoreNames.CAPTURE;
+
+    const settings: SettingMetadata[] | undefined = isCapture
+        ? [
+              {
+                  messageId: 'capturesTable.delete.removeCollectionsOption',
+                  setting: 'deleteAssociatedCollections',
+              },
+          ]
+        : undefined;
+
     return (
         <RowActionButton
-            confirmationMessage={<DeleteConfirmation />}
+            confirmationMessage={
+                <DeleteConfirmation
+                    messageId={
+                        isCapture ? 'capturesTable.delete.confirm' : null
+                    }
+                />
+            }
+            messageID="cta.delete"
             renderProgress={(item, index, onFinish) => (
                 <UpdateEntity
                     key={`Item-delete-${index}`}
+                    deleteCollections={isCapture}
```

Comment:
I think we're okay not passing this in. Since we are already passing in the store name I think we could do this check inside the component. Are we checking it here and passing it down since we are using it to set `settings`?

Reply:
No. Will remove.